modules/dbfs/mysql.go

```diff
@@ -368,8 +368,6 @@ func (di *DatabaseImpl) MySQLProjectRename(projectID int64, newName string) erro
 
 // MySQLProjectLookup returns the project name and permissions for a project with ProjectID = 'projectID'
 //
-// TODO (non-immediate/required): decide on change to MySQLProjectLookup stored proc
-// aka: decide if looking them up 1 at a time is good or not
 // Looking them up 1 at a time may seem worse, however we're looking up rows based on their primary key
```

Comment:
yeah, I actually thought I had removed this todo a while ago. I'm pretty convinced it's still the right way to be doing it